src/genericSchemeRegistry/schemes/StandardBounties.json

```diff
@@ -0,0 +1,682 @@
+{
+  "name": "Standard Bounties",
+  "addresses": {
+    "main": [
+      "0xa7135d0a62939501b5304a04bf00d1a9a22f6623"
+    ],
+    "rinkeby": [
+      "0x38f1886081759f7d352c28984908d04e8d2205a6"
+    ],
+    "private": [
+      "0x38f1886081759f7d352c28984908d04e8d2205a6"
+    ]
+  },
+  "actions": [
+    {
+      "id": "issueAndContribute",
+      "label": "Create and Fund Bounty",
+      "description": "Create a new bounty and contribute tokens to the bounty. IPFS upload of bounty data must adhere to the <a href=\"https://github.com/Bounties-Network/StandardBounties/blob/master/docs/standardSchemas.md\" target=\"_blank\">StandardBounties Schema</a> to be listed in Open Bounties.",
+      "notes": "https://github.com/Bounties-Network/StandardBounties/blob/master/contracts/StandardBounties.sol",
+      "fields": [
+        {
+          "label": "Sender Address",
+          "name": "_sender",
+          "placeholder": "Address (0x0000…)"
+        },
+        {
+          "label": "Issuer Addresses",
+          "name": "_issuers",
+          "placeholder": "DAO Avatar Address (0x0000…)"
+        },
+        {
+          "label": "Approver Addresses",
+          "name": "_approvers",
+          "placeholder": "DAO Avatar Address (0x0000…)"
+        },
+        {
+          "label": "Bounty Data",
+          "name": "_data",
+          "placeholder":"IPFS Hash (Qmd0...)"
+        },
+        {
+          "label": "Deadline",
+          "name": "_deadline",
+          "placeholder": "Unix Timestamp"
+        },
+        {
+          "label": "Token",
+          "name": "_token",
+          "defaultValue": "0x0000000000000000000000000000000000000000",
+          "placeholder": "Address (0x0000…)"
+        },
+        {
+          "label": "Token Version",
+          "name": "_tokenVersion",
+          "defaultValue": 0,
+          "placeholder":"0 for ETH, 20 for ERC20 Token"
+        },
+        {
+          "label": "Token Amount",
+          "name": "_depositAmount",
+          "decimals": 9,
```

Comment:
Please change this as I asked - we are using ETH as the unit everywhere in the application, not "GWei".

Reply:
I've reverted these fields to base unit of ETH as in a previous version. However, only whole units are passed, no fractions of ETH can be sent anymore. 8fa6cde